libs/auth-api-lib/src/lib/delegations/delegation-dto.mapper.ts

```diff
@@ -13,4 +16,14 @@ export class DelegationDTOMapper {
       scopes: dto.scopes,
     }
   }
+
+  public static recordToMergedDelegationDTO(
+    dto: DelegationRecordDTO,
+  ): MergedDelegationDTO {
+    return {
+      fromNationalId: dto.fromNationalId,
+      toNationalId: dto.toNationalId,
+      types: [dto.type as AuthDelegationType],
```

Comment:
If you type the `type` property of `DelegationRecordDTO` properly you don't need ot use `as`.

Reply:
Fixed.